src/components/ProjectName.tsx

```diff
@@ -17,6 +17,14 @@ export class ProjectName extends Component<Props, State> {
     fileName: this.props.value,
   };
   private handleBlur = (event: any) => {
+    let parent = (event.target as HTMLInputElement).parentElement;
+    while (parent) {
+      if (parent.tabIndex > -1) {
+        parent.focus();
+        break;
+      }
+      parent = parent.parentElement;
+    }
```

Comment:
this is just temp — we'll want to factor this out

Reply:
why not directly focus on modal?

Reply:
the ProjectName compo has no concept of a Modal. In fact it can live anywhere and shouldn't have that concept hardcoded.

Reply:
ok so basically we are trying to focus closest tabable parent, lets move it to a util then ?